### PR TITLE
added weapons button

### PR DESCRIPTION
**Pull request by ashwoodaa**

### CODE REVIEW

**Comment by MelanieRBaker** [comment]
:octocat:

**Comment by benwilburn** [comment]
👍

File: index.html [hunk 1]
```diff
@@ -144,18 +144,51 @@
...
-          <a class="card__back btn btn--small" 
+          <a class="card__back btn btn--small"
              href="#"
              previous="card--class">
             <span class="btn__prompt">&lt;</span>
             <span class="btn__text">Back</span>
-          </a>          
+          </a>
         </div>
       </div>
```
Comment (carter-harris) [inline index.html:157]: can you add a comment "chunk"

Comment (ashwoodaa) [inline index.html:157]: done

**Comment by carter-harris** [comment]
👍

**Pull request merged**